draft-ietf-tls-esni.md

```diff
@@ -226,8 +226,8 @@ The ECH configuration is defined by the following `ECHConfig` structure.
 
     struct {
         HpkeKeyConfig key_config;
-        uint16 maximum_name_length;
-        opaque public_name<1..2^16-1>;
+        uint8 maximum_name_length;
+        opaque public_name<1..255>;
```

Comment:
(2^16-1 in `public_name` is probably my fault. I bet I got it from HostName in RFC6066.)

Reply:
No one's fault :-)